lib/components/avatar.css

```diff
@@ -1,59 +1,47 @@
 .av {
-    height: 4.3rem;
-    width: 4.3rem;
+  height: 4.3rem;
+  width: 4.3rem;
 }
 
 .av,
 .av-sm,
-.av-md, 
-.av-lg, 
+.av-md,
+.av-lg,
 .av-xl {
-    border-radius: 100%;
+  border-radius: 100%;
 }
```

Comment:
`.av {`
`  height: 4.3rem;`
`  width: 4.3rem;`
` border-radius:100%`
`}`
Then add this class to all avatars, so you don't need to select all the avatar classes to add border radius.

Reply:
ok! got it
Thanks